docs/tutorials/postgresql.ipynb

```diff
@@ -0,0 +1,368 @@
+        "id": "xHxb-dlhMIzW"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "This tutorial shows how to create `tf.data.Dataset` from a PostgreSQL database server, so that the created `Dataset` could be passed to `tf.keras` for training or inference purposes.\n",
+        "\n",
+        "A SQL database is an important source of data for data scientist. As one of the most popular open source SQL database, [PostgreSQL](https://www.postgresql.org) is widely used in enterprises for storing critial and transactional data across the board. Creating `Dataset` from a PostgreSQL database server directly and pass the `Dataset` to `tf.keras` for training or inference, could greatly simplify the data pipeline and help data scientist to focus on building machine learning models."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "MUXex9ctTuDB"
+      },
+      "source": [
+        "## Setup and usage"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "upgCc3gXybsA",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install required tensorflow-io packages, and restart runtime"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "uUDYyMZRfkX4",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "%tensorflow_version 2.x \n",
+        "!pip install tensorflow-io-nightly"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yZmI7l_GykcW",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Install and setup PostgreSQL\n",
+        "\n",
+        "PostgreSQL database server is available on many platforms. In case it is not available, the following is needed to install PostgreSQL on Ubuntu:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "YUj0878jPyz7",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!apt-get install -y -qq postgresql\n",
+        "\n",
+        "!service postgresql start"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Afb01zkACsuW",
+        "colab_type": "text"
+      },
+      "source": [
+        "PostgreSQL will setup a user with username `postgres` by default. However, this username is not assigned a password initially. We will need to update the database first. The following assigns password `postgres` to username `postgres`:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "3HHfVdv9D3Jv",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!sudo -u postgres psql -U postgres -c \"ALTER USER postgres PASSWORD 'postgres';\""
```

Comment:
IDK postgress. But please confirm that we're not going to break someone's local installation if they run this notebook.

If there is a chance this will break people we need a big red warning at the top:

```
> Warning: *This notebook is designed to be run in a Google Colab only*. It installs packages on the system and requires root access. If you want to run it in a local Jupyter notebook, please proceed with caution.
```